composite/integration-tests/src/testShared/kotlin/com/apollographql/apollo3/internal/interceptor/ApolloServerInterceptorFileUploadTest.kt

```diff
@@ -181,7 +181,6 @@ class ApolloServerInterceptorFileUploadTest {
 
   @Test
   @Throws(Exception::class)
-  @Ignore("Creates a stack overflow at the moment")
```

Comment:
I forgot to enable this test while making the ResponseAdapters stateless. It's good to see it worked :)